storeshare_api/app/models/user.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
-
-
 class User < ApplicationRecord
+  has_many :subletters, dependent: :destroy
```

Comment:
should it be `has_one`? (i.e. exactly one subletter profile per user)

Reply:
shoot, I've missed it, making a correction now